packages/manager/tools/testcafe/.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Note for me:
Could be removed when we will be able to upgrade to ESLint@7.0.0 (https://eslint.org/blog/2020/05/eslint-v7.0.0-released#improved-developer-experience).